documentation/slate/source/includes/apis.md

```diff
@@ -1170,7 +1170,7 @@ direction       | int                    | Angle (in degree) between the previou
 
 |Field|Type|Description|
 |-----|----|-----------|
-|value|float|cost|
+|value|string|cost formatted as string|
```

Comment:
Isn't it necessary to mention that it is a 'float' cast into a 'string' ?

Reply:
We can. The current proposal is ambiguous enough to have integrator check the "real" response before coding :smile: (that's why I approved it).

Reply:
I'm just wondering what will happen if the integrator tries to cast the string in an 'int'

Reply:
After discussion: 'float' needs to be specified